sway-fmt-v2/src/items/item_fn/tests.rs

```diff
@@ -110,3 +110,37 @@ fmt_test!(  non_self_fn
         }
 }"
 );
+
+fmt_test!(  fn_with_nested_items
+"fn returns_msg_sender(expected_id: ContractId) -> bool {
+    let result: Result<Identity, AuthError> = msg_sender();
+    let mut ret = false;
+    if result.is_err() {
+        ret = false;
+    }
+    let unwrapped = result.unwrap();
+    match unwrapped {
+        Identity::ContractId(v) => {
+            ret = true
```

Comment:
Since we are not assigning to a variable from the match (instead we assign to `ret` from above) shouldn't the line end with `;`?

Reply:
I think in this instance the semicolon is optional, otherwise if it wasn't the test wouldn't pass